docs/src/content/docs/tr/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,294 @@
+- `colors.gray` — Arkaplan rengi ve kenarlıklar için kullanılır
+- `fontFamily.sans` — Kullanıcı arayüzü ve içerik metni için kullanılır
+- `fontFamily.mono` — Kod örnekleri için kullanılır
+
+```js {12,14,18,20}
+// tailwind.config.mjs
+import starlightPlugin from '@astrojs/starlight-tailwind';
+import colors from 'tailwindcss/colors';
+
+/** @type {import('tailwindcss').Config} */
+export default {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Tercih ettiğiniz vurgu rengi. Indigo, Starlight'ın varsayılana en yakın olanıdır.
+				accent: colors.indigo,
+				// Tercig ettiğiniz gri skalası. Zinc, Starlight'ın varsayılana en yakın olanıdır.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				//Tercih ettiğiniz yazı karakteri. Starlight, sistem yazı karakterlerini varsayılan olarak kullanır.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Tercih ettiğiniz kod yazı karakteri. Starlight sistem tek araklıklı yazı karakterlerini varsayılan olarak kullanır.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Temalandırma
+
+Starlight'ın renk teması, Starlight'ın varsayılan isteğe uyarlı özelliklerinin üzerine yazılmasıyla kontrol edilebilir.
+Bu değişkenler, metin ve arkaplan renkleri için gri gölgeler ve bağlantılar ve gezinti sırasındaki mevcut nesneler vurgulamak için vurgu rengi kullanıcı arayüzünce kullanılır.  
+
+### Renk teması editörü
+
+Starlight'ın vurgu ve gri renk paletlerini değiştirmek için aşağıdaki kaydırıcıları kullanın.
+Koyu ve açık önizleme alanları renkleri gösterecek ve ayrıca tüm sayfa değişiklerinizi önizlemek için güncellenecektir.
+
+Değişikliklerinizden memnun olduğunuzda, aşağıdaki CSS ya da Tailwind kodu kopyalayın ve projenizde kullanın.
+
+import ThemeDesigner from '~/components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Ön Ayarlar',
+			ocean: 'Okyanus',
+			forest: 'Orman',
+			oxide: 'Oksit',
+			nebula: 'Nebula',
+			default: 'Varsayılan',
+			random: 'Rasgele',
+		},
+		editor: {
+			accentColor: 'Vurgu',
+			grayColor: 'Gri',
+			hue: 'Ton',
+			chroma: 'Berraklık',
+			pickColor: 'Pick color',
+		},
+		preview: {
+			darkMode: 'Koyu mod',
+			lightMode: 'Açık Mod',
+			bodyText:
+				'Sayfa metni arkaplanla yüksek kontrastlı gri tonda görünür.',
+			linkText: 'Bağlantılar renklendirilmiştir.',
+			dimText: 'İçindekiler metni gibi bazı metinler daha düşük kontrasta sahiptir.',
+			inlineCode: 'Satır için kod belirgin arkaplana sahiptir.',
+		},
+	}}
+>
+	<Fragment slot="css-docs">
+	Bu temayı sitenize uygulamak için, [isteğe uyarlanmış CSS
+		dosyası içine](#özel-css-stilleri) aşağıdaki CSS'i dahil edin ve projenize ekleyin.
+	</Fragment>
+	<Fragment slot="tailwind-docs">
+	Aşağıdaki [Tailwind yapılandırma dosyasına ait](#tailwind-ile-starlightı-stillendirme) örnek, yapılandırma nesnesi içerisinde oluşturulmuş `accent` ve `gray` renk paletlerini dahil eder.
```

Comment:
I think this sentence is missing the `theme.extend.colors` reference that exists in the English version.

Reply:
Hello @HiDeoo , you are right! I've translated the sentence you mentioned.